README.md

```diff
@@ -1,102 +1,38 @@
 # TypeSpec
 
-[Try TypeSpec Online](https://aka.ms/trytypespec)
+[Official Docs](https://typespec.io/) | [Try TypeSpec Online](https://aka.ms/trytypespec) | [Getting Started](https://typespec.io/docs) | [Language Overview](https://typespec.io/docs/language-basics/overview)
 
-TypeSpec is a language for describing cloud service APIs and generating other API
-description languages, client and service code, documentation, and other assets.
-TypeSpec provides highly extensible core language primitives that can describe API
-shapes common among REST, OpenAPI, GraphQL, gRPC, and other protocols.
+TypeSpec is a language for defining cloud service APIs and shapes. TypeSpec is a highly extensible language with primitives that can describe API shapes common among REST, OpenAPI, GraphQL, gRPC, and other protocols.
 
-Using TypeSpec, you can create reusable patterns for all aspects of an API, along with the ability to check for and flag known anti-patterns. These patterns establish "guardrails" for API designers and make it easier to follow best practices than deviate from them. TypeSpec promotes highly regular API designs that adhere to best practices by construction.
+TypeSpec is excellent for generating many different API description formats, client and service code, documentation, and many other assets. All this while keeping your TypeSpec definition as a single source of truth.
```

Comment:
While I support dropping the overemphasis on guardrails, I think it is very important to mention TypeSpec's ability to encapsulate and compose APIs from components.